# ðŸŒ¾*Oryza sativa*[æ°´ç¨»]
+ åŸºå› ç»„[ã€ŠA Draft Sequence of the Rice Genome (Oryza sativa L. ssp. indica)ã€‹](http://science.sciencemag.org/content/296/5565/79) - 430 Mb
+ å¶ç»¿ä½“[NC_001320.1](https://www.ncbi.nlm.nih.gov/nuccore/11466763) - 134525 bp
+ çº¿ç²’ä½“[NC_007886.1](https://www.ncbi.nlm.nih.gov/nuccore/NC_007886.1) - 491515 bp

## é¡¹ç›®1ä¿¡æ¯
+ [ã€ŠGenome sequencing of rice subspecies and genetic analysis of recombinant lines reveals regional yield- and quality-associated lociã€‹](https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-018-0572-x#Sec1)
+ PRJNA486425
+ Oryza sativa indica[ç±¼ç¨»] * Oryza sativa japonica[ç²³ç¨»]
+ Recombinant inbred lines (RILs) derived from a cross between SN265 and R99 (indica) 

> + ç±¼ç¨»çš„ç±³ç²’æ·€ç²‰ç²˜æ€§è¾ƒå¼±ï¼Œèƒ€æ€§è¾ƒå¤§ï¼Œè°·ç²’ç‹­é•¿ï¼Œé¢–æ¯›çŸ­ç¨€ï¼Œå¶ç»¿ã€è‰²è¾ƒæ·¡ï¼Œå¶é¢å¤šèŒ¸æ¯›ï¼Œè€è‚¥æ€§è¾ƒå¼±ï¼Œå¶ç‰‡å¼¯é•¿ï¼Œæ ªåž‹è¾ƒæ¾æ•£ï¼Œå¹¶æœ‰è€æ¹¿ã€è€çƒ­ã€è€å¼ºå…‰ã€æ˜“è½ç²’å’Œå¯¹ç¨»ç˜Ÿç—…æŠµæŠ—æ€§è¾ƒå¼ºç­‰ç‰¹å¾ç‰¹æ€§ã€‚
> + ç²³ç¨»çš„ç±³ç²’æ·€ç²‰ç²˜æ€§è¾ƒå¼ºï¼Œèƒ€æ€§è¾ƒå°ï¼Œè°·ç²’çŸ­åœ†ï¼Œé¢–æ¯›é•¿å¯†ï¼Œå¶ç»¿ã€è‰²è¾ƒæµ“ï¼Œå¶é¢è¾ƒå…‰æ»‘ï¼Œè€è‚¥æ€§è¾ƒå¼ºï¼Œå¶ç‰‡çŸ­ç›´ï¼Œæ ªåž‹ç´§å‡‘ï¼Œå¹¶æœ‰è€å¯’ã€è€å¼±å…‰ã€ä¸æ˜“è½ç²’å’Œå¯¹ç¨»ç˜Ÿç—…æŠµæŠ—æ€§è¾ƒå¼±ç­‰ç‰¹å¾ç‰¹æ€§

## [é¡¹ç›®1Runä¿¡æ¯](https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=PRJNA486425&go=go)


| type | No | szie.bp | depth |
| --- | --- | --- | --- |
| parent(male)     | SRR7750269 | 6,521,979,122 * 2 | 30 |
| parent(Female)   | SRR7750271 | 6,687,761,564 * 2 | 30 |
| filialness(R155) | SRR7750270 |
| filialness(R81) | SRR7750272 |
| filialness(R82) | SRR7750273 |
| filialness(R8) | SRR7750274 |
| filialness(R77) | SRR7750275 |
| filialness(R78) | SRR7750276 |
| filialness(R79) | SRR7750277 |
| filialness(R80) | SRR7750278 |
| filialness(R73) | SRR7750279 |

## 2. é¡¹ç›®2ä¿¡æ¯
+ PRJNA232554
+ [NATURE ã€ŠParentâ€“progeny sequencing indicates higher mutation rates in heterozygotesã€‹](https://www.nature.com/articles/nature14649)
+ [ã€ŠWidely distributed hot and cold spots in meiotic recombination as shown by the sequencing of rice F2 plantsã€‹](https://nph.onlinelibrary.wiley.com/doi/full/10.1111/nph.13319)

### 2.1 ç›¸å…³ä¿¡æ¯
+ LYP9ï¼šä¸¤ä¼˜åŸ¹ä¹ï¼ˆåŽŸå65002ï¼‰å³LYP9ï¼Œæ˜¯æ±Ÿè‹çœå†œä¸šç§‘å­¦é™¢é‚¹æ±ŸçŸ³ç ”ç©¶å‘˜ç­‰ä¸“å®¶è‚²æˆçš„ä¸¤ç³»ç±¼åž‹æ‚äº¤ç¨»æ–°ç»„åˆã€‚
+ ç›®å‰é™¤äº†èº«ä¸ºä¹¡é•‡å†œæŠ€å‘˜å‰æ–°è€˜ç­‰ç›åŸŽå†œæ°‘è®¤ä¸ºâ€œä¸¤ä¼˜åŸ¹ä¹â€æ˜¯ä¸€èµ·â€œç§‘æŠ€é€ å‡äº‹ä»¶â€ï¼Œå¹¶è¯‰ä¹‹å…¬å ‚å¤–ï¼Œæ±Ÿè‹çœå†œç§‘é™¢åŽŸå‰¯ç ”ç©¶å‘˜æž—é•¿å¹³ä¹Ÿæ›¾è´­ä¹°ä¸¤ä¼˜åŸ¹ä¹è¿›è¡Œç”Ÿäº§ï¼Œç»“æžœäºæŸ300å¤šä¸‡å…ƒã€‚ç›®å‰æž—å·²å‘å›½å®¶ä¸“åˆ©å±€æå‡ºâ€œæ’¤é”€ä¸¤ä¼˜åŸ¹ä¹å›½å®¶ä¸“åˆ©â€çš„ç”³è¯·

### 2.3 Runä¿¡æ¯
| Relationship | Type | No | Source | SRR | Size | Depth |
| --- | --- | --- | --- | --- | --- | --- |
| maternal | Oryza sativa L. cultivars | PA64 | Dr Cailin Wang at the Institute of Food Crops, Jiangsu Academy of Agricultural Sciences, China | SRR1060330 | 
| paternal | Oryza sativa L. cultivars | 9311 | Dr Cailin Wang at the Institute of Food Crops, Jiangsu Academy of Agricultural Sciences, China | SRR1060331 | 
| F1 | hybrid rice | LYP9 | grow | SRR1060365	| 
| F2 | hybrid rice | LYP9F2_22 | grow | SRR1060368 |



## parent

### SRR7750269
+ 30
```bash
WORKING_DIR=~/stq/data/anchr/Oryza_sativa/Recombination
BASE_NAME=SRR7750269
cd ${WORKING_DIR}/${BASE_NAME}
bash 0_realClean.sh

anchr template \
    . \
    --basename ${BASE_NAME} \
    --queue mpi \
    --genome 1_000_000 \
    --fastqc \
    --kmergenie \
    --insertsize \
    --sgapreqc \
    --trim2 "--dedupe --cutoff 120 --cutk 31" \
    --qual2 "25" \
    --len2 "60" \
    --filter "adapter,phix,artifact" \
    --mergereads \
    --ecphase "1,2,3" \
    --cov2 "40 80 120 160 240 320" \
    --tadpole \
    --splitp 100 \
    --statp 1 \
    --fillanchor \
    --xmx 110g \
    --parallel 24

# æäº¤è¶…ç®—ä»»åŠ¡
bsub -q mpi -n 24 -J "${BASE_NAME}" "
  bash 0_master.sh
  cd ${WORKING_DIR}/${BASE_NAME}
  if [ -d ./align ];
  then
    echo -n
  else
    mkdir ./align
  fi
  ~/stq/Applications/biosoft/bwa-0.7.13/bwa mem \
      -t 20 \
      -M   \
      ../genome/genome.fa \
      ./2_illumina/trim/Q25L60/R1.fq.gz \
      ./2_illumina/trim/Q25L60/R2.fq.gz > ./align/Rp.sam
  ~/stq/Applications/biosoft/bwa-0.7.13/bwa mem \
      -t 20 \
      -M   \
      ../genome/genome.fa \
      ./2_illumina/trim/Q25L60/Rs.fq.gz > ./align/Rs.sam
      
  cp ./align/Rp.sam ./align/R.sam
  cat ./align/Rs.sam | grep -v "^@" >> ./align/R.sam
  samtools view -b -o ./align/R.bam ./align/R.sam
  samtools sort -o ./align/R.sort.bam ./align/R.bam
  samtools index ./align/R.sort.bam
"
```

### 
```bash
WORKING_DIR=~/stq/data/anchr/Oryza_sativa/Recombination
BASE_NAME=SRR7750271
cd ${WORKING_DIR}/${BASE_NAME}
bash 0_realClean.sh

anchr template \
    . \
    --basename ${BASE_NAME} \
    --queue mpi \
    --genome 1_000_000 \
    --fastqc \
    --kmergenie \
    --insertsize \
    --sgapreqc \
    --trim2 "--dedupe --cutoff 120 --cutk 31" \
    --qual2 "25" \
    --len2 "60" \
    --filter "adapter,phix,artifact" \
    --mergereads \
    --ecphase "1,2,3" \
    --cov2 "40 80 120 160 240 320" \
    --tadpole \
    --splitp 100 \
    --statp 1 \
    --fillanchor \
    --xmx 110g \
    --parallel 24

# æäº¤è¶…ç®—ä»»åŠ¡
bsub -q mpi -n 24 -J "${BASE_NAME}" "
  bash 0_master.sh
  cd ${WORKING_DIR}/${BASE_NAME}
  if [ -d ./align ];
  then
    echo -n
  else
    mkdir ./align
  fi
  ~/stq/Applications/biosoft/bwa-0.7.13/bwa mem \
      -t 20 \
      -M   \
      ../genome/genome.fa \
      ./2_illumina/trim/Q25L60/R1.fq.gz \
      ./2_illumina/trim/Q25L60/R2.fq.gz > ./align/Rp.sam
  ~/stq/Applications/biosoft/bwa-0.7.13/bwa mem \
      -t 20 \
      -M   \
      ../genome/genome.fa \
      ./2_illumina/trim/Q25L60/Rs.fq.gz > ./align/Rs.sam
      
  cp ./align/Rp.sam ./align/R.sam
  cat ./align/Rs.sam | grep -v "^@" >> ./align/R.sam
  samtools view -b -o ./align/R.bam ./align/R.sam
  samtools sort -o ./align/R.sort.bam ./align/R.bam
  samtools index ./align/R.sort.bam
"
```

## filialness
```bash
list=($(seq 69 79))
WORKING_DIR=~/stq/data/anchr/Oryza_sativa/Recombination
BASE_NAME=SRR77502
for i in ${list[@]};
do
  if [ ${i} -eq 69 ] || [ ${i} -eq 71 ];
  then
    echo -n
  else
    cd ${WORKING_DIR}/${BASE_NAME}${i}
    bsub -q mpi -n 24 -J "${i}" "
    if [ -d ./align ];
      then
        echo -n
      else
        mkdir ./align
      fi
      ~/stq/Applications/biosoft/bwa-0.7.13/bwa mem \
          -t 20 \
          -M   \
          ../genome/genome.fa \
          ./2_illumina/R1.fq.gz \
          ./2_illumina/R2.fq.gz > ./align/Rp.sam
          
      cp ./align/Rp.sam ./align/R.sam
      cat ./align/Rs.sam | grep -v "^@" >> ./align/R.sam
      samtools view -b -o ./align/R.bam ./align/R.sam
      samtools sort -o ./align/R.sort.bam ./align/R.bam
      samtools index ./align/R.sort.bam
    "
  fi
done
```
